src/GraphQL/Execution/DocumentExecuter.cs

```diff
@@ -147,7 +149,10 @@ private void ValidateOptions(ExecutionOptions options)
                 {
                     using (metrics.Subject("schema", "Initializing schema"))
                     {
-                        config.FieldMiddleware.ApplyTo(config.Schema);
+                        if (config.EnableLogging)
```

Comment:
I think this needs a different flag.  Disabling field middleware is different than disabling the metrics gathering.  You can use field middleware for more than just metrics.

Reply:
Yeah i like this idea much better, this change was pushed.